genesyscloud/routing_queue/resource_genesyscloud_routing_queue.go

```diff
@@ -1030,9 +1030,15 @@ func updateQueueMembers(d *schema.ResourceData, sdkConfig *platformclientv2.Conf
 	if len(newUserIds) > 0 {
 		log.Printf("Sleeping for 10 seconds")
 		time.Sleep(10 * time.Second)
+
+		members, diagErr := getRoutingQueueMembers(d.Id(), "group", sdkConfig)
+		if diagErr != nil {
+			return diagErr
+		}
+
 		for _, userId := range newUserIds {
-			if err := verifyUserIsNotGroupMemberOfQueue(d.Id(), userId, sdkConfig); err != nil {
-				log.Println(err.Error())
+			if err := verifyUserIsNotGroupMemberOfQueue(d.Id(), userId, members); err != nil {
+				return diag.FromErr(err)
```

Comment:
Do we want to use one of our builder functions here for errors.

Reply:
Yes, I'll fix that